Singularity/Singularity/Units/JobType.cs

```diff
@@ -1,14 +1,9 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace Singularity.Units
+﻿namespace Singularity.Units
 {
     public enum JobType
```

Comment:
Im a little confused. Why do we have jobs AND Jobtypes. I thought jobs are just there to identify what a unit is doing right now (therefore jobtypes alone would be enough).

Reply:
You're right. just created another PR with the removal of Job.cs